project.clj

```diff
@@ -98,7 +98,7 @@
                        ;; https://github.com/district0x/district-server-db/blob/3839edd/project.clj#L12
                        ;; After deployment changes, (transitive) dependencies are not installed automatically
                        ;; TODO: remove, they should be installed automatically
-                       [better-sqlite3 "5.4.0"]
+                       [better-sqlite3 "4.0.0"]
```

Comment:
You can add a comment that this needs to be 4.0 because we need to use Node 8 - also mention it in README setup .

Reply:
Actually I think I'll revert this, because Node 8 is too old for some packages and causes issues. It's quite terrible but user should probably just switch to Node 8 and older better-sqlite, if they're heading for a deploy. One more reason to go for the truffle deploy.